datalad/tests/test_cmd.py

```diff
@@ -285,6 +285,22 @@ def test_runner_failure_unicode(path):
         runner.run(u"β-command-doesnt-exist", cwd=path)
 
 
+@skip_if_on_windows  # likely would fail
+@with_tempfile(mkdir=True)
+def test_runner_fix_PWD(path):
+    env = os.environ.copy()
+    env['PWD'] = orig_cwd = os.getcwd()
+    runner = Runner(cwd=path, env=env)
+    out, err = runner.run(
+        ['python', '-c', 'import os; print(os.environ["PWD"])'],
```

Comment:
Elsewhere we've avoided assuming `python` is available (see #2287).  How about using `sys.executable` instead?

Reply:
right! that was the intention - will fix up now, thanks!